### PR TITLE
[BOLT][DWARF][NFC] Added double escape characters

### PR DESCRIPTION
**Pull request by sayhaan**
Added double escape characters to lines that describe a test.

### CODE REVIEW

**Comment by dwblaikie** [comment]
(personally - I'm not sure if this convention is really a great one, especially unsure whether it's worthwhile mass-migrating code to use it... (seems like it's pretty close to other broad formatting changes which we don't usually apply broadly) - before any of these patches go in, maybe worth a discourse thread?)

**Comment by ayermolo** [comment]
> (personally - I'm not sure if this convention is really a great one, especially unsure whether it's worthwhile mass-migrating code to use it... (seems like it's pretty close to other broad formatting changes which we don't usually apply broadly) - before any of these patches go in, maybe worth a discourse thread?)

@sayhaan  is my intern and I asked him to do it as a startup task primarily to get familiar with the PR workflow (thus only BOLT tests). :)
Personally I don't have a strong opinion about it, but figure might as well since it seems like a current convention.

**Comment by dwblaikie** [comment]
Fair enough. If you folks (bolt owners) are on board, carry on

**Comment by ayermolo** [comment]
@sayhaan Please update summary removing internal task number reference.

File: bolt/test/X86/jmp-optimization.test [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Tests the optimization of functions that just do a tail call in the beginning.
+## Tests the optimization of functions that just do a tail call in the beginning.
 
 # This test has commands that rely on shell capabilities that won't execute
```
Comment (ayermolo) [inline bolt/test/X86/jmp-optimization.test:3]: This also

File: bolt/test/X86/split-landing-pad.s [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+## function optimization is enabled. In particular, function foo is splitted
+## to two fragments:
+##    foo: contains 2 try blocks, which invokes bar to throw exception
+##    foo.cold.1: contains 2 corresponding catch blocks (landing pad)
+##
+## Similar to split jump table, split landing pad target to different fragment.
+## This test is written to ensure BOLT safely handle these targets, e.g., by
+## marking them as non-simple.
 #
 # Steps to write this test:
```
Comment (ayermolo) [inline bolt/test/X86/split-landing-pad.s:11]: here also

File: bolt/test/no-relocs.test [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Verifies that input without relocations is rejected in relocs mode.
+## Verifies that input without relocations is rejected in relocs mode.
 
 # This test uses the clang driver without target flags and will only succeed
```
Comment (ayermolo) [inline bolt/test/no-relocs.test:3]: Here also

File: bolt/test/pie.test [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Check that we do not reject position-independent executables (PIEs).
+## Check that we do not reject position-independent executables (PIEs).
 
 # This test uses the clang driver without target flags and will only succeed
```
Comment (ayermolo) [inline bolt/test/pie.test:3]: here

File: bolt/test/re-optimize.test [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Check that we detect re-optimization attempt.
+## Check that we detect re-optimization attempt.
 
 # This test uses the clang driver without target flags and will only succeed
```
Comment (ayermolo) [inline bolt/test/re-optimize.test:3]: here

File: bolt/test/shared-object.test [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# Test that llvm-bolt processes *.so without a failure
+## Test that llvm-bolt processes *.so without a failure
 
 # This test uses the clang driver without target flags and will only succeed
```
Comment (ayermolo) [inline bolt/test/shared-object.test:3]: here

**Comment by dwblaikie** [comment]
> @sayhaan Please update summary removing internal task number reference.

FWIW, I think we did land on a policy where it's acceptable (though I'm not suggesting to encourage it) to include internal bug/issue/etc references in commit messages (so long as they aren't used as a substitute for comprehensive explanations that anyone can read upstream) but not in source code.

File: bolt/test/X86/patch-entries.test [hunk 7]
```diff
@@ -10,22 +10,22 @@ RUN:   -Wl,-q -I%p/../Inputs
...
+## Check the expected eh_frame contents
 RUN: llvm-nm --print-size %t.out > %t.foo
 RUN: llvm-objdump %t.out --dwarf=frames >> %t.foo
 RUN: FileCheck %s --input-file %t.foo --check-prefix=CHECK-FOO
 CHECK-FOO: 0000000000[[#%x,FOO:]] [[#%x,OPTSIZE:]] t foo
 CHECK-FOO: 0000000000[[#%x,ORG:]] [[#%x,ORGSIZE:]] t foo.org.0
-# patched FDE comes first
+## patched FDE comes first
 CHECK-FOO: FDE {{.*}} pc=00[[#%x,ORG]]...00[[#%x,ORG+ORGSIZE]]
 # original FDE comes second
```
Comment (ayermolo) [inline bolt/test/X86/patch-entries.test:30]: here also

**Comment by ayermolo** [comment]
Looks like one of the tests is failing, please check it's not caused by your changes

**Review by ayermolo — APPROVED** [review]

**Pull request merged**